core/trino-main/src/main/java/io/trino/server/security/oauth2/ScribeJavaOAuth2Client.java

```diff
@@ -84,7 +84,7 @@ public DynamicCallbackOAuth2Service(OAuth2Config config, HttpClient httpClient)
                     config.getClientId(),
                     config.getClientSecret(),
                     null,
-                    Joiner.on(",").join(config.getScopes()),
+                    Joiner.on(" ").join(config.getScopes()),
```

Comment:
Can you please add a test with multiple scopes that would fail without your change?